src/tincctl.c

```diff
@@ -1519,7 +1519,10 @@ static int cmd_pcap(int argc, char *argv[]) {
 static void sigint_handler(int sig) {
 	(void)sig;
 
-	fprintf(stderr, "\n");
+	if(write(2, "\n", 2) < 0) {
```

Comment:
Should be 1, not 2.

Reply:
Oh rats, I made a mess of my own. Thanks for the review. FWIW, `STDERR_FILENO` doesn't work here because of msvc.